Library/Homebrew/test/support/helper/spec/shared_context/integration_test.rb

```diff
@@ -126,8 +126,8 @@ def setup_test_formula(name, content = nil)
       content = <<~EOS
         desc "Some test"
         homepage "https://example.com/#{name}"
-        url "file://#{TEST_FIXTURE_DIR}/tarballs/testball-0.1.tbz"
-        sha256 "#{TESTBALL_SHA256}"
+        url "file://#{TEST_FIXTURE_DIR}/tarballs/#{OS.linux? ? "testball-0.1-linux.tbz" : "testball-0.1.tbz"}"
+        sha256 "#{OS.linux? ? TESTBALL_SHA256_LINUX : TESTBALL_SHA256}"
```

Comment:
Can this use the same approach as above?

Reply:
No. This code defines the location of the tarball and its checksum, which is platform dependent.
```ruby
Formulary.factory("testball1").stable.checksum.hexdigest
```

Reply:
It could calculate the checksum from the contents of the file, rather than store it in Ruby code.

Reply:
> It could calculate the checksum from the contents of the file, rather than store it in Ruby code.

Yes, let's do this.

Reply:
Done!